NEWS

```diff
@@ -58,8 +58,34 @@ included in the vX.Y.Z section and be denoted as:
    (** also appeared: A.B.C)  -- indicating that this item was previously
                                  included in release version vA.B.C.
 
-4.1.3 -- TBD
-------------
+4.1.3 -- March, 2022
+--------------------
+
+- Minor datatype performance improvements in the CUDA-based code paths.q
+- Fix MPI_ALLTOALLV when used with MPI_IN_PLACE.
+- Fix MPI_BOTTOM handling for non-blocking collectives.  Thanks to
+  Lisandro Dalcin for reporting the problem.
+- Enable OPAL memory hooks by default for UCX.
+- Many compiler warnings fixes, particularly for newer versions of
+  GCC.
+- Fix intercommunicator overflow with large payload collectives.  Also
+  fixed MPI_REDUCE_SCATTER_BLOCK for similar issues with large payload
+  collectives.
+- Back-port ROMIO 3.3 fix to use stat64() instead of stat() on GPFS.
+- Fixed several non-blocking MPI collectives to not round fractions
+  based on float precision.
+- Fix compile failure for --enable-heterogeneous.  Also updated the
+  README to clarify that --enable-heterogeneous is functional, but
+  still not recomended for most environments.
+- Minor fixes to OMPIO, including:
+  - Fixing the open behavior of shared memory shared file pointers.
```

Comment:
- Fixing the open behavior of shared file pointers

Reply:
actually, leave it as it is, it is ok.

Reply:
@edgargabriel and I conversed in Slack: we updated the text and I re-pushed.